open_event/static/js/admin/event/details.js

```diff
@@ -0,0 +1,33 @@
+$( document ).ready(function() {
+    $('#wizard').smartWizard({
+        includeFinishButton:false,
+        keyNavigation:false,
+        noForwardJumping: true,
+    });
+
+    $('.date-picker').daterangepicker({
+          singleDatePicker: true,
+          timePicker: true,
+          calender_style: "picker_4",
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/triplez/open-event-orga-server/file/2664119187/issues/source?bid=3233864&fileBranchId=3324174#l11)